tools/libboard/board_dc/board_dc.h

```diff
@@ -97,6 +97,29 @@ fpga_result parse_fw_ver(char *buf, char *fw_ver, size_t len);
 fpga_result read_mac_info(fpga_token token, uint32_t afu_channel_num,
 				struct ether_addr *mac_addr);
 
+/**
+* Prints Security information.
+*
+* @param[in] token            fpga_token object for device (FPGA_DEVICE type)
+* @returns FPGA_OK on success. FPGA_NOT_FOUND if MAX10 or NIOS sysfs not found.
+* FPGA_INVALID_PARAM if invalid parameters were provided
+*/
+fpga_result print_sec_info(fpga_token token);
+
+/**
+* Get sysfs value.
+*
+* @param[in] token           fpga_token object for device (FPGA_DEVICE type)
+* @param[in] bmcfw_var       pointer to sysfs path
+* @param[inout] len          returns sysfs value as string
+
+* @returns FPGA_OK on success. FPGA_NOT_FOUND if invalid MAC address.
+* FPGA_INVALID_PARAM if invalid parameters were provided
+*
+*/
+fpga_result read_sysfs(fpga_token token, char *sysfs_path, char *sysfs_name);
```

Comment:
Should this go into a generic library that can share the function across each of your libboard_X implementations?

Reply:
moved common code libborad_common.so file